src/qutip_cupy/dense_functions.py

```diff
@@ -29,3 +29,32 @@ def trace_cupydense(cp_arr):
     # @TODO: whnen qutip allows it we should remove this call to item()
     # as it takes a time penalty commmunicating data from GPU to CPU.
     return cp.trace(cp_arr._cp).item()
+
+
+hermdiff_kernel = cp.RawKernel(
```

Comment:
nitpick: I am not sure if this is a public or private module. If it's public, then it's better not exposed. 
```suggestion
_hermdiff_kernel = cp.RawKernel(
```